File: Unity/Assets/Eruru.MVVM/Demo/Scripts/View Models/ViewModel.cs
using UnityEngine;

namespace Eruru.MVVM.Demo {

	class ViewModel {

		public User User {

			get {
				return _User;
			}

			set {
				_User = value;
			}

		}
		public MVVMRelayCommand OnClick { get; set; }

		User _User = new User ();

		public ViewModel () {
			OnClick = new MVVMRelayCommand (value => Debug.Log (User.Account));
		}

	}

}

File: IntelliJ IDEA/Java/src/demo/clickButton/ViewModel.java
package demo.clickButton;

import com.eruru.mvvm.MVVMRelayCommand;

import javax.swing.*;

public class ViewModel {

	private MVVMRelayCommand onClick = new MVVMRelayCommand (value -> {
		JOptionPane.showMessageDialog (null, "You clicked this button");
	});

	public MVVMRelayCommand getOnClick () {
		return onClick;
	}

	public void setOnClick (MVVMRelayCommand onClick) {
		this.onClick = onClick;
	}

}